src/applications/vaos/components/WaitTimeAlert.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { Link } from 'react-router';
+import { MENTAL_HEALTH } from '../utils/constants';
+import newAppointmentFlow from '../newAppointmentFlow';
+
+export const WaitTimeAlert = ({
+  preferredDate,
+  nextAvailableApptDate,
+  typeOfCareId,
+  eligibleForRequests,
+  onClickRequest,
+}) => {
+  const today = moment();
+  const momentPreferredDate = moment(preferredDate);
+
+  const actions = (
+    <div className="vads-u-display--flex vads-u-margin-top--2 vads-u-align-items--center">
+      {eligibleForRequests && (
+        <>
+          <Link to={newAppointmentFlow.requestDateTime.url}>
+            <button className="usa-button-secondary" onClick={onClickRequest}>
+              Request earlier appointment
+            </button>
+          </Link>
+          <span className="vads-u-display--inline-block vads-u-margin-x--2 vads-u-font-weight--bold">
+            OR
+          </span>
+        </>
+      )}
+      <span>Call your VA facility directly</span>
+    </div>
+  );
+
+  // If preferred date is today, show urgent care warning
+  if (today.isSame(momentPreferredDate, 'day')) {
```

Comment:
Aren't we still defaulting the preferred date to today? Won't this display when they first open the page?

Reply:
Unsetting default preferred date to match mockups which should resolve this.